src/bootctl/bootctl-install.c

```diff
@@ -1039,9 +1039,10 @@ int verb_install(int argc, char *argv[], void *userdata) {
         graceful = arg_graceful() == ARG_GRACEFUL_FORCE || (!install && arg_graceful() != ARG_GRACEFUL_NO);
 
 #if HAVE_OPENSSL
+        _cleanup_(openssl_ask_password_ui_freep) OpenSSLAskPasswordUI *ui = NULL;
```

Comment:
this really needs a comment, that makes clear that this apparently load bearing. Otherwise the next one touching this will remove this again, and again, and again.

i don't really understand why it is load bearing though. can you explain? best would be in a comment